src/js/actions/layers.js

```diff
@@ -768,55 +769,87 @@ define(function (require, exports) {
             return Promise.resolve();
         }
 
+        if (modifier !== undefined && layerSpec.size > 1) {
```

Comment:
In particular, if you don't pass a modifier, no pivot is set. This is the case when we call select internally (i.e., not from the layer face's click handler.)